SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/Spatial2WorkerMap.cpp

```diff
@@ -5,11 +5,13 @@
 #include "SpatialGDKFunctionalTests/SpatialGDK/CrossServerAndClientOrchestrationTest/CrossServerAndClientOrchestrationTest.h"
 #include "SpatialGDKFunctionalTests/SpatialGDK/RegisterAutoDestroyActorsTest/RegisterAutoDestroyActorsTest.h"
 #include "SpatialGDKFunctionalTests/SpatialGDK/RelevancyTest/RelevancyTest.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/SpatialTestMultiServerUnrealComponents/SpatialTestMultiServerUnrealComponents.h"
 #include "TestWorkerSettings.h"
 
 USpatial2WorkerMap::USpatial2WorkerMap()
 	: UGeneratedTestMap(EMapCategory::CI_PREMERGE_SPATIAL_ONLY, TEXT("Spatial2WorkerMap"))
 {
+	SetNumberOfClients(2);
```

Comment:
Well, theoretically this is not neccessary, since the default setting is 2 for our tests, but I guess it's okay to specify it explicitly if a test needs it.

Reply:
Oh I didn't know that. Seems better to be explicit about it though, but happy to remove if you prefer.

Reply:
I prefer explicit to be honest.

Reply:
Yeah, I think explicit is better, especially if a test like this actually does require 2 clients.